### ISSUE TITLE
My tflite file cannot run in your YOLOv9

### ISSUE THREAD

**Issue by lucky-baby-dog** [surendramaran/YOLO#12]
![屏幕截图 2024-10-23 094550](https://github.com/user-attachments/assets/e58fe65c-44a7-4763-98fc-e20079ebc605)

![屏幕截图 2024-10-23 094724](https://github.com/user-attachments/assets/786062a8-c011-471b-9522-3fc1c04cdebe)
![屏幕截图 2024-10-23 094753](https://github.com/user-attachments/assets/d133c207-717d-4f33-b574-64827b068320)
Hello, I changed v9 in your file to my model file, but when I recompile and run it again, it doesn't open. What's going on

**Comment by lucky-baby-dog** [comment]
![屏幕截图 2024-10-23 100857](https://github.com/user-attachments/assets/9284cb36-57a4-4121-9ad3-b9f2b6f82b9f)
My tflite file can be opened in Netron software and displayed

**Comment by surendramaran** [comment]
Any error it is showing in logcat when you try to run?
If not then in which way the model is behaving?

**Comment by lucky-baby-dog** [comment]
Brother, the problem lies here. How can we solve it

![屏幕截图 2024-10-23 173145](https://github.com/user-attachments/assets/a3609009-9edf-4f7f-96a2-0f4f2493849a)

**Comment by lucky-baby-dog** [comment]
> 我可以知道输出和输入的形状吗？访问https://netron.app/并获取属性。

Sure, how should I send it to you

**Comment by lucky-baby-dog** [comment]
![屏幕截图 2024-10-23 181701](https://github.com/user-attachments/assets/1fceabea-ff38-4fe2-9953-8bfd4e715d04)

**Comment by lucky-baby-dog** [comment]
> May I know the output and input shape. visit https://netron.app/ and get properties.

![屏幕截图 2024-10-23 181701](https://github.com/user-attachments/assets/2a2a0e82-2b14-4311-b7a5-c38cf953ee2f)

**Comment by surendramaran** [comment]
Yeah, this YOLOv9 model is probably exported and trained from here https://github.com/WongKinYiu/yolov9

While I took models from here https://docs.ultralytics.com/models/yolov9/

So, you may also use  ultralytics's environment, I will also try to interpret your model, if successful I will publish codes.

**Comment by surendramaran** [comment]
You mind sharing your  tflite model, with labels and sample images?

**Comment by lucky-baby-dog** [comment]
> You mind sharing your tflite model, with labels and sample images? [surendramaran8@gmail.com](mailto:surendramaran8@gmail.com)

Hello, I have packaged the items and placed them in the data file，My email address is majiajun518@gmail.com

**Comment by yg-dickson** [comment]
Does it support yolo11?

**Comment by lucky-baby-dog** [comment]
> Does it support yolo11?

I haven't tried it before, you can give it a try

**Comment by Woo-wdy** [comment]
yolo11模型部署到yolov8上直接闪退，我用的是yolo11x的模型转成了yolo11_float16.tflite,把他放在yolov8或者yolov9的android里并调用会直接闪退

**Comment by Woo-wdy** [comment]
我找到了原因 把GPU加速关掉就好了  但是识别速度会很慢

**Comment by lucky-baby-dog** [comment]
> 我找到了原因 把GPU加速关掉就好了 但是识别速度会很慢



> 我找到了原因 把GPU加速关掉就好了 但是识别速度会很慢

I switched to my YOLOV9 model and ran it inside YOLOV9, but it crashed and the software couldn't open

**Comment by LYS86** [comment]
> 我找到了原因 把GPU加速关掉就好了 但是识别速度会很慢

插个眼，使用GPU去推理yolo10也会崩溃，且崩溃无法拦截

**Comment by Adamya113** [comment]
#Reason: 
Ultralytics includes metadata along with the tflite model, which is used by android studio (https://ai.google.dev/edge/litert/models/metadata)

#Solution: 
1. Either you can use the ultralytics export facility for conversion 
```
    from ultralytics import YOLO

    # Load a model
    model = YOLO("yolo11n.pt")  # load an official model
    model = YOLO("path/to/best.pt")  # load a custom trained model

    # Export the model
    model.export(format="REQUIRED FORMAT")
    ```

2. You can add the metadata yourself: (line 1325) https://github.com/ultralytics/ultralytics/blob/main/ultralytics/engine/exporter.py
